### ISSUE TITLE
Create Windows Executable

### ISSUE THREAD

**Issue by rivermont** [rivermont/spidy#47]
Most likely using something like [py2exe](http://www.py2exe.org/).

**Comment by kneekill** [comment]
I can do this if it's still up for grabs.

**Comment by mdaizovi** [comment]
Is this still up? I've used py2exe before, could take a look.

**Comment by rivermont** [comment]
@mdaizovi Of course! I'm not sure whether @kneekill's made any progress, but we can't have too many attempts if so.

**Comment by kneekill** [comment]
Sorry for the radio silence on this, I've been absolutely swamped lately
and haven't had the chance to finish this. Feel free to take it @mdaizovi

On Oct 24, 2017 6:23 AM, "Will Bennett" <notifications@github.com> wrote:

@mdaizovi <https://github.com/mdaizovi> Of course! I'm not sure whether
@kneekill <https://github.com/kneekill>'s made any progress, but we can't
have too many attempts if so.

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/rivermont/spidy/issues/47#issuecomment-338988285>, or mute
the thread
